app/uk/gov/hmrc/lightweightcontactevents/controllers/CreationController.scala

```diff
@@ -44,15 +48,15 @@ class CreationController @Inject()() extends BaseController {
   def create(): Action[AnyContent] = Action.async {implicit request =>
     createContact(request.body.asJson) match {
       case Right(contact) => {
-//        val email = Email(contact, init)
-//        val result: Future[Try[Int]] = emailConnector.sendEmail(email)
-//        result map {
-//          case Success(s) =>
-//            Ok
-//          case Failure(ex) =>
-//            Logger.warn("Sending contact email fails with message " + ex.getMessage)
-//            BadRequest("Sending contact email fails with message " + ex.getMessage)
-//          }
+        val jsonData = VOADataTransfer(contact, init)
+        val result: Future[Try[Int]] = dataTransferConnector.transfer(jsonData)
+        result map {
+          case Success(s) =>
+            Ok
+          case Failure(ex) =>
+            Logger.warn("Sending contact email fails with message " + ex.getMessage)
+            BadRequest("Sending contact email fails with message " + ex.getMessage)
+          }
         Future.successful(Ok)
```

Comment:
There is no need for this line although it might not be executed, it was there just to simulate a VOA endpoint which wasn't present in the past